go/updater/updater_windows.go

```diff
@@ -21,7 +21,7 @@ func openApplication(applicationPath string) error {
 
 func (u *Updater) applyUpdate(localPath string) (tmpPath string, err error) {
 	if strings.HasSuffix(localPath, ".exe") {
-		err = exec.Command(localPath).Start()
+		err = exec.Command(localPath, "/SILENT").Start()
```

Comment:
If we ever want to switch to an installer .exe that has problems with this command line option, I guess we could bridge it with an intermediate release.